### PR TITLE
Define extension capabilities

### PR DESCRIPTION
**Pull request by andreastt**
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/343)

### CODE REVIEW

**Comment by shs96c** [comment]
I need to do a review, but capabilities should be in the firstMatch or alwaysMatch fields, not as new top level properties of the new session command.

File: webdriver-spec.html [hunk 1]
```diff
@@ -1426,16 +1426,68 @@
...
+<p>As with <a>extension commands</a>,
+ it is suggested that the key used to denote
+ the <a>extension capability</a> namespace
+ is based on the <a href=http://www.w3.org/TR/CSS21/syndata.html#vendor-keywords>vendor keywords</a>
+ listed in [[!CSS21]]
+ and preceeds the first "<code>:</code>" character in the string.
+
+<aside class=example>
+ <p><a>Extension capabilities</a> are typically used
+  to provide UA- or <a>intermediary node</a> specific configuration
```
Comment (shs96c) [inline webdriver-spec.html]: s/UA-/UA/

File: webdriver-spec.html [hunk 2]
```diff
@@ -1426,16 +1426,68 @@
...
+  that is not handled by the <a>table of capabilities</a>.
+
+ <p>An example <a>new session</a> request body
+  might look like this:
+
+ <pre>{
+	"alwaysMatch": …,
+	"firstMatch": …,
```
Comment (shs96c) [inline webdriver-spec.html]: Missing opening brace.

Comment (andreastt) [inline webdriver-spec.html]: There is an opening brace on the `<pre>` line.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: Fixed.

**Comment by andreastt** [comment]
> I need to do a review, but capabilities should be in the firstMatch or alwaysMatch fields, not as new top level properties of the new session command. 

I think I will fix this by not saying explicitly where it should be.

**Comment by shs96c** [comment]
LGTM. Merging.

**Pull request merged**